staging/src/k8s.io/sample-controller/artifacts/examples/crd-status-subresource.yaml

```diff
@@ -1,13 +1,35 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: foos.samplecontroller.k8s.io
+  annotations:
+    "api-approved.kubernetes.io": "unapproved, experimental-only"
 spec:
   group: samplecontroller.k8s.io
-  version: v1alpha1
+  versions:
+    - name: v1
```

Comment:
Why don't we keep the group version `v1alpha1`? In that way the generated files won't get changed

Reply:
that's a good point, but I don't think these examples should be the place where we encourage CRD authors to develop stable API.

a CRD author should start with an alpha API, and gradually evolve the API into a stable one using [CRD versioning](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/). This example misses that journey. 

another nit is this CRD has a `"api-approved.kubernetes.io": "unapproved, experimental-only"`, so probably leaving it as `v1alpha1` makes more sense.

Reply:
> a CRD author should start with an alpha API, and gradually evolve the API into a stable one using CRD versioning. This example misses that journey.

Got it 👍🏻 
> another nit is this CRD has a "api-approved.kubernetes.io": "unapproved, experimental-only", so probably leaving it as v1alpha1 makes more sense.

iirc, the k8s.io annotation is a requirement of the API version itself and not the group version?